xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -91,17 +84,17 @@ static void fetchDisplayModes()
         s_res_cur_displayMode.iWidth = s_res_cur_displayMode.iScreenWidth = m.getPhysicalWidth();
         s_res_cur_displayMode.iHeight = s_res_cur_displayMode.iScreenHeight = m.getPhysicalHeight();
         s_res_cur_displayMode.fRefreshRate = m.getRefreshRate();
-        s_res_cur_displayMode.dwFlags= D3DPRESENTFLAG_PROGRESSIVE;
-        s_res_cur_displayMode.bFullScreen   = true;
+        s_res_cur_displayMode.dwFlags = D3DPRESENTFLAG_PROGRESSIVE;
+        s_res_cur_displayMode.bFullScreen = true;
         s_res_cur_displayMode.iSubtitles = s_res_cur_displayMode.iHeight;
-        s_res_cur_displayMode.fPixelRatio   = 1.0f;
+        s_res_cur_displayMode.fPixelRatio = 1.0f;
         s_res_cur_displayMode.strMode = StringUtils::Format(
             "{}x{} @ {:.6f}{} - Full Screen", s_res_cur_displayMode.iScreenWidth,
             s_res_cur_displayMode.iScreenHeight, s_res_cur_displayMode.fRefreshRate,
             s_res_cur_displayMode.dwFlags & D3DPRESENTFLAG_INTERLACED ? "i" : "");
 
         std::vector<CJNIDisplayMode> modes = display.getSupportedModes();
-        for (auto m : modes)
+        for (CJNIDisplayMode& m : modes)
```

Comment:
```suggestion
        for (const CJNIDisplayMode& m : modes)
```
?

Reply:
No, since the CJNIDisplayMode functions used below are not marked const

Reply:
hum to me it seems they could be marked const as they are simple getters